### DISCUSSION TITLE
Just a chat

### DISCUSSION THREAD

**Discussion by EmiOnGit** [EmiOnGit/warbler_grass#23] [General]
Can be used to just communicate about stuff outside of issues etc.

**Comment by EmiOnGit** [comment]
Btw @janhohenheim , I'd still like to have some form of communication about what we are implementing to be more in sync. 
I'd suggest still using PRs for that, do you have another idea?
However let's not care to much about trivial changes in this regard

**Reply by janhohenheim** [reply]
Agreed, I don't like to merge stuff without a second pair of eyes checking what I did anyways!
Plus, PRs let us generate nice changelogs.

**Reply by EmiOnGit** [reply]
Nice. I'm currently working on a naive version for the heightmap implementation, to have an idea how that might look. 
After that I suggest to move towards a release on crates.io to reserve the name. 
This would mean to cleanup everything and make sure it's at least useable in the release state.
What do you think?

**Reply by janhohenheim** [reply]
Agreed, although you might want to reconsider the crate name. `warblersneeds` to me reads like a single fictional word, so I had some trouble remembering the name until I realized it was supposed to be read as `warblers_needs` 🙂

**Reply by EmiOnGit** [reply]
Uh haven't thought about, that since the name was always obvious to me ( well who might have guessed ).
To change the name seems reasonable and now would be the time to do so

**Reply by EmiOnGit** [reply]
However the name is just a name I came up with spontaneous I am also open to a completely new name.
May I ask for your honest thoughts on the name?

**Comment by janhohenheim** [comment]
In the generator I found this line:
```rust
height_deviation = (height_deviation - 0.5) * 2. * generator_config.height_deviation;
```
What's the reasoning behind the 2 in this transform? I know `height_deviation` is ~ U(0, 1), so this looks like a kind of z-transform, since E[`height_deviation`] = 0.5.
Is `generator_config.height_deviation` the standard deviation or the mean absolute deviation?

**Reply by EmiOnGit** [reply]
Was supposed to be the mean absolute deviation, this is misleading as you'd expect standard deviation.
However, I'd suggest reworking or just deleting the complete generator part of the project soon
